skyportal/handlers/api/observingrun.py

```diff
@@ -162,7 +162,14 @@ def get(self, run_id=None):
             return self.success(data=data)
 
         runs = ObservingRun.query.order_by(ObservingRun.calendar_date.asc()).all()
-        return self.success(data=runs)
+        runs_list = []
+        for run in runs:
+            runs_list.append(run.to_dict())
+            runs_list[-1]["run_end_utc"] = run.instrument.telescope.next_sunrise(
```

Comment:
This isn't when the run ends, but the next sunrise on the telescope. For the sunrise of the run, I think you want `run.sunrise`, but check to make sure

Reply:
@dannygoldstein I don't see a `sunrise` field defined on ObservingRun in models.py. There is only `rise_time()` which only reports rise times for specific targets passed in to the function. I can add a `sunrise` property, but what would be the best way to compute that? So I don't want to go through the telescope instance?

Reply:
I take this back - this is fine. Sorry about this. IDK what I was thinking here ...